pyanaconda/core/product.py

```diff
@@ -15,17 +15,10 @@
 # License and may only be used or replicated with the express permission of
 # Red Hat, Inc.
 #
-import configparser
 import os
-from collections import namedtuple
 from functools import cache
 
-ProductData = namedtuple("ProductData", [
-    "is_final_release",
-    "name",
-    "version",
-    "short_name",
-])
+from pyanaconda.modules.common.structures.product import ProductData
```

Comment:
You are importing the structure definition, you instead want to use the dbus API to read from the module the values.

from pyanaconda.modules.runtime.user_interface.ui_interface import UIInterface

Reply:
You need to define the UI_INTERFACE here: pyanaconda/modules/common/constants/objects.py 

Then get the proxy so that you can read values from there: 
proxy = RUNTIME.get_proxy(UI_INTERFACE)

Then you can read product_data like:
product_data = proxy.ProductData